web/src/components/users/UserEditForm.vue

```diff
@@ -86,7 +87,7 @@
           :model-value="groupMembershipAttributes.divisionId"
           :type="UserGroupTypes.DIVISION"
           :parent-id="groupMembershipAttributes.departmentId"
-          :disabled="isNil(groupMembershipAttributes.departmentId)"
+          :disabled="isNil(groupMembershipAttributes.departmentId) || !isSystemAdmin"
```

Comment:
I would generally put policy check first.
```suggestion
          :disabled="!isSystemAdmin ||isNil(groupMembershipAttributes.departmentId)"
```